templates/CRM/Report/Form/Layout/Table.tpl

```diff
@@ -110,9 +110,10 @@
                 {foreach from=$columnHeaders item=header key=field}
                     {assign var=fieldLink value=$field|cat:"_link"}
                     {assign var=fieldHover value=$field|cat:"_hover"}
+                    {assign var=fieldClass value=$field|cat:"_class"}
                     <td class="crm-report-{$field}{if $header.type eq 1024 OR $header.type eq 1 OR $header.type eq 512} report-contents-right{elseif $row.$field eq 'Subtotal'} report-label{/if}">
                         {if $row.$fieldLink}
-                            <a title="{$row.$fieldHover}" href="{$row.$fieldLink}">
+                            <a title="{$row.$fieldHover}" href="{$row.$fieldLink}"  {if $row.$fieldClass} class="{$row.$fieldClass}"{/if}>
```

Comment:
@eileenmcnaughton won't there always be ```$row.$fieldClass``` so to omit IF condition, as we are already ensuring the fieldName to be present at L115?

Reply:
hmm but if empty it would be class="" - would that be ok?

Reply:
if empty then we don't have that link at all

Reply:
So the data would look like this
```
$row => array(
  'financial_type_id' => 'Donation',
  'financial_type_id_link' => 'https....',
  'contact_type' => 'Individual',
  'contact_type_link' => 'https....',
  'contact_type_hover' => 'Click to popup contact',
  'contact_type_class' => 'crm-pop-up',
);




```